templates/ga-release-post.adoc

```diff
@@ -115,26 +115,36 @@ Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Libe
 
 == Run your apps using RELEASE_VERSION
 
-If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], include the following in your `pom.xml` file:
 
 [source,xml]
 ----
-<dependency>
-    <groupId>io.openliberty</groupId>
-    <artifactId>openliberty-runtime</artifactId>
-    <version>RELEASE_VERSION</version>
-    <type>zip</type>
-</dependency>
+<plugin>
+    <groupId>io.openliberty.tools</groupId>
+    <artifactId>liberty-maven-plugin</artifactId>
+    <version>3.8.2</version>
+</plugin>
 ----
 
-Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle], include the following in your `build.gradle` file:
 
 [source,gradle]
 ----
-dependencies {
-    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[RELEASE_VERSION,)'
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.6.2'
+    }
```

Comment:
What about the portion the applies the Liberty Gradle plugin? That is also necessary.

`apply plugin: 'liberty'`

Reply:
How should this be added in the scope of the above?  Could you use the 'Add a suggestion' to add it how you believe it should be presented to the end user?